sites/dotnet-guide/contents/tutorials/web-fundamentals/introduction/index.md

```diff
@@ -0,0 +1,32 @@
+---
+type: TutorialStep
+date: 2020-12-01
+title: Introduction
+technologies: [asp.net, javascript]
+topics: [web]
+products: [rider]
+author: maartenba
+subtitle: Introduction to the Web Fundamentals in Rider series.
+thumbnail: ./thumbnail.png
+longVideo: 
+    poster: ./poster.png
+    url: https://youtu.be/todo
+---
+
+While [Rider is a .NET IDE](https://www.jetbrains.com/rider/), it integrates other tools such as [DataGrip](https://www.jetbrains.com/datagrip/) for database development, and [WebStorm](https://www.jetbrains.com/webstorm/) for web development.
+
+You can download these tools separately, but it's not required - they are already part of Rider!
+
+The **Web Fundamentals in Rider** tutorial covers front-end development in Rider, showing capabilities that are brought in by WebStorm.
+It covers navigation, coding assistance, refactoring, running, debugging and testing your web applications, with technologies like JavaScript and React.
+
+Tutorial steps make use of the [sample code](https://TODO_PAUL) that was created using `npx create-react-app`.
```

Comment:
TODO_Paul @pauleveritt

Reply:
(see issue opening mention of this specific TODO :) I will update once the link is there)